src/main/scala/coreplex/BaseCoreplex.scala

```diff
@@ -32,7 +32,7 @@ trait HasCoreplexParameters {
   lazy val lsb = p(BankIdLSB)
   lazy val innerParams = p.alterPartial({ case TLId => "L1toL2" })
   lazy val outermostParams = p.alterPartial({ case TLId => "Outermost" })
-  lazy val outermostMMIOParams = p.alterPartial({ case TLId => "MMIO_Outermost" })
+  lazy val outerMMIOParams = p.alterPartial({ case TLId => "L2toMMIO" })
```

Comment:
Yeah, I noticed that later on. I think they should both be outerMMIOParams, since they are at the same level as outerParams (with TLKey "L2toMC").